### PR TITLE
Fixed footer spacing.

### PR DESCRIPTION
**Pull request by Anita-ihuman**
**Description**
Adjusted the padding spaces to fit Figma design.
Screenshot 2021-06-26 at 13 39 05

This PR fixes #

**Notes for Reviewers**
This issue was fixed before and the footer list looked clustered..

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: docs/_sass/footer.scss [hunk 1]
```diff
@@ -288,7 +290,7 @@ footer {
   .footer-columns {
     display: flex;
     justify-content: center;
-    // padding-right: 37px;
+    // padding-right: 43px;
```
Comment (vinayaksh42) [inline docs/_sass/footer.scss:293]: @Anita-ihuman, this change is not affecting the final styling. It can be removed.

**Review by vinayaksh42 — CHANGES_REQUESTED** [review]

File: docs/_sass/footer.scss (hunk 1 above)
Comment (Anita-ihuman) [reply docs/_sass/footer.scss]: I have fixed this

**Comment by vinayaksh42** [comment]
LGTM! @Anita-ihuman  nice work

**Comment by leecalcote** [comment]
Thank you, @Anita-ihuman and @vinayaksh42

**Pull request merged**